packages/flutter/test/material/app_bar_theme_test.dart

```diff
@@ -0,0 +1,106 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('AppBar theme does not affect defaults', (WidgetTester tester) async {
+    await tester.pumpWidget(MaterialApp(
+      home: Scaffold(appBar: AppBar()),
+    ));
+
+    final Material widget = _getAppBarMaterial(tester);
+
+    expect(widget.color, Colors.blue);
+    expect(widget.elevation, equals(4.0));
+  });
+
+  testWidgets('AppBar theme overrides color', (WidgetTester tester) async {
+    const Color themedColor = Colors.lightBlue;
+    const AppBarTheme theme = AppBarTheme(color: themedColor);
+
+    await tester.pumpWidget(MaterialApp(
+      theme: ThemeData(appBarTheme: theme),
+      home: Scaffold(appBar: AppBar()),
+    ));
+
+    final Material widget = _getAppBarMaterial(tester);
+    expect(widget.color, themedColor);
+  });
+
+  testWidgets('AppBar color - Widget', (WidgetTester tester) async {
+    const Color themeColor = Colors.white;
```

Comment:
Are some of these here because they can't be null? If so, put in a comment. I kept trying to figure out if you meant to check their values against something.

Reply:
Or just pass the values instead of making a const.

Reply:
You're right, I just passed the values directly instead of a const now